crowbar_framework/app/models/node_object.rb

```diff
@@ -724,7 +724,12 @@ def rebuild_run_list
     crowbar["run_list_map"] = {} if crowbar["run_list_map"].nil?
 
     # Cull by state
-    map = crowbar["run_list_map"].select { |k,v| v["states"].include?("all") or v["states"].include?(self.crowbar["state"]) }
+    node_state = self.crowbar['state']
```

Comment:
Style/RedundantSelf: Redundant `self` detected. (https://github.com/bbatsov/ruby-style-guide#no-self-unless-required)<br>Style/StringLiterals: Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping. (https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)